package.json

```diff
@@ -213,7 +213,6 @@
     "chalk": "1.1.3",
     "cheerio": "0.22.0",
     "concurrently": "3.1.0",
-    "csp-parse": "0.0.2",
```

Comment:
`csp-parse` was lower casing the entire CSP header string which was breaking the tests for the SHA since they're case sensitive. There was a PR on the repo with no response from the maintainer. Turns out parsing the header really isn't very hard so I wrote one.